guides/common/Makefile

```diff
@@ -66,6 +66,8 @@ ccutil:
 		ccutil compile --lang=en-US --format html-single ; \
 	fi
 
+toc:
+
```

Comment:
Since it's empty, can we drop it?
```suggestion
```

Reply:
No, we can't, the global task calls it:

Output after removing it:

```
.... snip ....

make -C "doc-Upgrading_Project_Disconnected" html
make[2]: Entering directory '/mnt/work/guides/doc-Upgrading_Project_Disconnected'
bundle exec sass --sourcemap=none --no-cache --style compressed -I ../common/css ../common/css/default.scss ../build/Upgrading_Project_Disconnected/katello.css
bundle exec asciidoctor -a build=katello  -a docinfo=shared -a docinfodir=common -a date_mdy="May 23 2024" -a date_my="May 2024" -a docdatetime="May 2024" -a stylesheet=../build/Upgrading_Project_Disconnected/katello.css -a attribute-missing=warn -b html5 -d book --trace -o ../build/Upgrading_Project_Disconnected/index-katello.html master.adoc 2>&1 | tee ../build/Upgrading_Project_Disconnected/index-katello.html.log 1>&2
CHECKING FOR ASCIIDOCTOR ERRORS AND WARNINGS
CHECKING FOR ORPHAN XREFS HTML LINKS
make[2]: Leaving directory '/mnt/work/guides/doc-Upgrading_Project_Disconnected'
make[1]: Leaving directory '/mnt/work/guides'
make -C guides/ toc
make[1]: Entering directory '/mnt/work/guides'
make -C "doc-Administering_Project" toc
make[2]: Entering directory '/mnt/work/guides/doc-Administering_Project'
make[2]: *** No rule to make target 'toc'.  Stop.
make[2]: Leaving directory '/mnt/work/guides/doc-Administering_Project'
make[1]: *** [Makefile:13: doc-Administering_Project] Error 2
make[1]: Leaving directory '/mnt/work/guides'
make: *** [Makefile:44: toc] Error 2
```

Reply:
Oh yes, we have
```make
$(SUBDIRS):
	$(MAKE) -C "$@" $(MAKECMDGOALS)
```